frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts

```diff
@@ -365,9 +365,10 @@ export class CastDragAndDrop implements OnInit {
     this.castPositions[positionIndex].dancerCount += 1;
   }
 
-  private changeCastCount(change: number) {
+/** Increments cast count by given amount, pass in negative value to decrement. */
+private incrementCastCountUtil(amount: number) {
```

Comment:
Why the 'util" suffix? It seems redundant.

Reply:
There is already and incrementCastCount function in the system.